src/lib/util/util.ts

```diff
@@ -682,7 +682,12 @@ export const shuffle = <T>(array: T[]): T[] => {
 	return array;
 };
 
-export const random = (num: number) => Math.round(Math.random() * num);
+export const random = (num: number) => Math.floor(Math.random() * num);
+
+export const randomBetween = (a: number, b: number) => {
+	const [minimum, maximum] = a > b ? [b, a] : [a, b];
+	return random(maximum - minimum) + minimum;
+};
```

Comment:
test coverage goes ⬇️ 

That is to say, We have a test for `random` wherein `Math.random` is mocked. So this should be tested also.

Reply:
That's left-over from earlier iterations.